.github/workflows/tests.yml

```diff
@@ -57,9 +57,9 @@ jobs:
             -   name: Configure composer
                 run: |
                     ([ -d ~/.composer ] || mkdir ~/.composer) && cp .github/composer-config.json ~/.composer/config.json
-                    SYMFONY_VERSION=$(cat composer.json | grep '^ *\"branch-version\". *\"[1-9]' | grep -o '[0-9.]*')
-                    echo "::set-env name=SYMFONY_VERSION::$SYMFONY_VERSION"
-                    echo "::set-env name=COMPOSER_ROOT_VERSION::$SYMFONY_VERSION.x-dev"
+                    SYMFONY_VERSION=$(cat composer.json | grep '"branch-version":\s*"[0-9]*\.[0-9]*"' | grep -o '[0-9]*\.[0-9]*')
```

Comment:
I'd prefer not changing this line unless it needs to be

Reply:
Ah yes, that happens when working on two CI tasks in parallel. The old expression did not work on the Mac build. But here, this is an unrelated change, so I'm going to revert the line.